app/routes/circulars/circulars.server.ts

```diff
@@ -262,3 +279,156 @@ export async function circularRedirect(query: string) {
     throw redirect(circularURL)
   }
 }
+
+export async function syncSynonyms({ synonyms }: { synonyms: string[] }) {
+  if (synonyms.length === 0) {
+    return
+  }
+  const db = await tables()
+  const doc = db._doc as unknown as DynamoDBDocument
+
+  const tableName = db.name('synonyms')
+
+  doc.put({
+    TableName: tableName,
+    Item: {
+      synonyms: synonyms.join(','),
+    },
+  })
+}
```

Comment:
```suggestion
  await db.synonyms.put({
    TableName: tableName,
    Item: {
      synonyms: synonyms.join(','),
    },
  })
}
```

Reply:
According to the app.arc file, the partition key of this table is called `id`, but this field is not specified here.